src/test/java/com/robosh/ejournal/service/AdminServiceTest.java

```diff
@@ -52,7 +52,6 @@ void Should_ThrowValidationException_WhenPasswordNotEquals_ForSaveAdmin() {
         adminToSave.setConfirmedPassword("notsamepassword");
         ValidationException validationException = assertThrows(ValidationException.class, () -> adminService.save(adminToSave));
         assertEquals("Password should be same", validationException.getMessage());
```

Comment:
should not you extract it to constant?

Reply:
I don't see needs to do that now, but if bigger part of the team won't I will do that.